crates/bevy_input/src/keyboard.rs

```diff
@@ -31,206 +46,365 @@ pub fn keyboard_input_system(
     }
 }
 
-/// The key code of a keyboard input.
+/// The key code of a [`KeyboardInput`](crate::keyboard::KeyboardInput).
+///
+/// ## Usage
+///
+/// It is used as the generic `T` value of an [`Input`](crate::Input) to create a `bevy`
```

Comment:
is the name of the resource missing here? What's a `bevy` resource?

Reply:
No. A `bevy` resource is anything that can be used with `Res<Type>` inside of a system. This essentially tells you that we create a `bevy` resource using `KeyCode` as the generic `T` of `Input` creating the `Input<KeyCode>` resource that can be accessed using `Res<Input<KeyCode>>`.

Reply:
In that case, I think I would say "to create a `Res<Input<KeyCode>>`".